src/app/recording/Recording.js

```diff
@@ -331,6 +323,16 @@ function Recording () {
     setSelectedSegmentId(createdFeature.id)
   }
 
+  async function onBoundsChange(bounds) {
+    const topRight = `${bounds._northEast.lng},${bounds._northEast.lat}`
+    const bottomRight = `${bounds._southWest.lng},${bounds._northEast.lat}`
+    const bottomLeft = `${bounds._southWest.lng},${bounds._southWest.lat}`
+    const topLeft = `${bounds._northEast.lng},${bounds._southWest.lat}`
+    const boundingBox = `${topRight},${bottomRight},${bottomLeft},${topLeft},${topRight}`
```

Comment:
So the bounding box is actually an arbitrary shape? Does it mean you can also retrieve data for single districts and Kiezes?

If yes: cool. 
But also: It makes it a little harder to track which areas were already requested. So from frontend side I'd stick we should stick with rectangular shapes for the moment.